spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryConverter.java

```diff
@@ -243,4 +242,13 @@ private void convertProteinExistence(
             document.proteinExistence = proteinExistence.getId();
         }
     }
+
+    private void populateSpellcheckSuggestions(UniProtDocument document) {
+        // populate fields used in spellcheck
+        populateSuggestions(document.proteinNames, document);
+        populateSuggestions(document.geneNamesExact, document);
+        Collection<String> diseases = document.commentMap.getOrDefault("cc_disease", List.of());
+        populateSuggestions(diseases, document);
+        populateSuggestions(document.rcStrain, document);
+    }
```

Comment:
hmm.. I am not sure how to say that taxon suggestion is in converter. Could you please put comment about that here.

Reply:
Yes, i saw that. I was saying that the suggestion population logic is not at one place. Better to tell the reader of the code to look at Taxonomy suggestion also. Yes, tests will help us. But sometimes, we don't run the tests.

Reply:
I am adding the comment, I just do not see this comment important, because I would try findUsage in populateSuggestions method, and it would show all places that is being used.